### ISSUE TITLE
This project works on the ESP32 too?

### ISSUE THREAD

**Issue by beyonlo** [anson-vandoren/esp8266-captive-portal#11]
Hello, I would like to have a captive portal on my ESP32. My intention is in really that when mobile user acess de ESP32 Access Point without password, can open automatically the default browser on the SO (Android/IOS) with the IP of ESP32 device, without user needs to type the ESP32 IP on the browser.

I think that the captive portal will works for this purpose, right?

Ps: Mobile user do not need access the internet via ESP32, just access automatically the ESP32 HTTP page on it.

Thank you!

**Comment by anson-vandoren** [comment]
Yes, this should also work on the ESP-32 same as on the ESP-8266
